src/main/java/com/global/inbox/dto/ItemDto.java

```diff
@@ -0,0 +1,15 @@
+package com.global.inbox.dto;
+
+import com.sun.istack.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class ItemDto {
```

Comment:
Seems, like it doesn't have all the required information to describe an item. Also, if we are using layer with DTOs (that's completely fine and remind me next meeting I can give a bit more overview) then there is one more good tool that can be used for it. `mapstruct` (we can have this integration later)